src/h5web/dataset-visualizer/DatasetVisualizer.tsx

```diff
@@ -44,15 +45,24 @@ function DatasetVisualizer(props: Props): ReactElement {
     });
   }
 
-  // Fetch dataset value asynchronously
   const { getValue } = useContext(ProviderContext);
+  const [scheduleLoadingFlag, cancelLoadingFlag] = useTimeoutFn(() => {
+    mergeState({ loading: true });
+  }, 50);
```

Comment:
I do not get why you need to use a function with a timeout. Is it to skip the loading part if it goes fast enough ?

Reply:
Ok my bad. Next time, I will read the PR description :see_no_evil: